Fix TODOs for Community Guide (#246)
scientific-python/learn.scientific-python.org: content/community/role.md

  and loyal user community.
- A community manager plays a critical role in building and sustaining an active
  and vibrant community around the project. They organize and promote community
  events, facilitate discussions, and encourage knowledge-sharing. This helps
  to foster connections, drive participation, and create a positive and
  collaborative atmosphere within the community.
- A community manager serves as a trusted resource for community members,
  offering support, answering questions, and resolving conflicts. They act as a
  point of contact, providing guidance, feedback, and assistance when needed.
- A community manager helps to promote the project's values, goals, and mission.
  They advocate for the project and its benefits, both within the existing user
  community and to potential new users. This can lead to increased visibility,
  growth, and adoption of the project.
- A community manager plays a crucial role in gathering user feedback and
  insights. They act as the voice of the community, conveying user needs,
  preferences, and concerns to the project team.

## How to find community manager for your project?

> - Find what do you need exactly - Determine scope, figuring out roles and

Reviewer: ```suggestion
- Define what your project needs, specifically - Determine scope, figuring out roles and
```

  responsibilities for the role

> - Familiarity with the domain of the project and science

Reviewer: ```suggestion
- Familiarity with the science domain of the project (if relevant)
```
